### DISCUSSION TITLE
Rad Pro: Reset history/settings

### DISCUSSION THREAD

**Discussion by Dsl71** [Gissio/radpro#41] [Ideas]
Settings to default and erase all settings and data logging flash space.

**Comment by Gissio** [comment]
Isn't it easier to reflash the device?

**Comment by Dsl71** [comment]
nope, for that you have to open it again. But this has only low priorty.

**Comment by Gissio** [comment]
I’ve added data log reset to the latest beta (using a trick to minimize flash memory wear).

https://github.com/Gissio/radpro/releases/tag/2.0.1beta3

Could you test this?

**Reply by Dsl71** [reply]
will do soon ;-)
